storage/tokudb/tokudb_thread.h

```diff
@@ -34,6 +34,58 @@ Copyright (c) 2006, 2015, Percona and/or its affiliates. All rights reserved.
 namespace tokudb {
 namespace thread {
 
+#if (defined(__MACH__) || defined(__APPLE__)) && _POSIX_TIMERS <= 0
+
+#define _x_min(a, b) ((a) < (b) ? (a) : (b))
+
+#define timed_lock_define(timed_func_name, lock_type_name, lock_func_name)     \
+inline                                                                         \
+int                                                                            \
+timed_func_name(lock_type_name *mutex,                                         \
+                     const struct timespec *abs_timeout) {                     \
```

Comment:
This is not quite the coding style that we are following for TokuDB. It should look more like:
inline int timed_func_name(
    lock_type_name* mutex,
    const struct timespec* abs_timeout) {